packages/ui/src/components/va-backtop/VaBacktop.stories.ts

```diff
@@ -1,12 +1,79 @@
-import { defineComponent } from 'vue'
-import VaBacktop from './VaBacktop.demo.vue'
+import { VaBacktop } from './'
+import { VaRadio } from '../va-radio'
 
 export default {
   title: 'VaBacktop',
   component: VaBacktop,
 }
 
-export const Default = defineComponent({
+export const Default = () => ({
```

Comment:
I think we need an interaction test for this story (all others are fine without).